src/language-js/parser-babel.js

```diff
@@ -16,22 +16,17 @@ function babelOptions({ sourceType, extraPlugins = [] }) {
     errorRecovery: true,
     createParenthesizedExpressions: true,
     plugins: [
+      // When adding a plugin, please add a test in `tests/js/babel-plugins`,
+      // To remove plugins, remove it here and run `yarn test tests/js/babel-plugins` to verify
+
       "doExpressions",
-      "objectRestSpread",
       "classProperties",
       "exportDefaultFrom",
-      "exportNamespaceFrom",
-      "asyncGenerators",
       "functionBind",
       "functionSent",
-      "dynamicImport",
+      // "dynamicImport", // Need ref
```

Comment:
Can't find a link says this enabled, but removing it seems no effect, the ast is also right 

```text
          "callee": {
            "type": "Import",
            "start": 0,
            "end": 6,
            "loc": {
              "start": { "line": 1, "column": 0 },
              "end": { "line": 1, "column": 6 }
            }
          },
```